_includes/trusted.html

```diff
@@ -0,0 +1,50 @@
+<style type="text/css" scoped>
+  .trusted-section {
+    color: #eaf1f7;
+    background-color: #2c3e50;
+    padding: 4em;
+  }
+
+  .trusted-container {
+    max-width: 1600px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 1.5em;
+  }
+
+  .logo-container {
+    justify-content: space-around;
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .client-logo {
+    margin-right: 12px;
+    max-height: 68px;
+    padding: 8px;
+  }
+
+  .client-logo.svg {
+    padding: 16px 8px;
+  }
+
+  @media screen and (max-width: 767px) {
+    .trusted-section {
+      padding-top: 3em;
+      padding-bottom: 3em;
+    }
+  }
+</style>
+
+<section class="trusted-section">
+  <div class="trusted-container">
+    <h4>deranged is trusted by:</h4>
+    <a href="/cases" class="logo-container">
+      <img src="/img/marmar.png" class="client-logo" alt="" />
+      <img src="/img/fischer-lighting.png" class="client-logo" alt="" />
+      <img src="/img/kbh-stadsarkiv.png" class="client-logo" alt="" />
+      <img src="/img/nornorm-km.png" class="client-logo" alt="" />
```

Comment:
The non-png images we could probably improve performance on by using an img srcset and doing webp versions for browsers that support it? Also differenty sizes etc.